src/store/modules/backend.js

```diff
@@ -2,9 +2,9 @@
 import Vue from 'vue';
 import { times } from 'lodash-es';
 // eslint-disable-next-line import/no-cycle
-import Backend from '../../utils/backend';
 import { createDeepLinkUrl } from '../../utils';
 
+// todo: getters
```

Comment:
Can you remove this comment?

If the PR is still work in progress you can mark it as draft.

Reply:
sure

Reply:
no it's not WIP. but I will remove all useless comments.

Reply:
I just mention that https://github.com/aeternity/superhero-ui/issues/992 in this comment

Reply:
@mradkov done, all useless comments are removed. anyway I have an issues for the same propose.